### PR TITLE
Fixed windows segfault

### PR DESCRIPTION
**Pull request by xarkes**

### CODE REVIEW

File: libr/cons/cons.c [hunk 1]
```diff
@@ -465,7 +465,7 @@ R_API void r_cons_clear_line(int std_err) {
 		char white[1024];
 		memset (&white, ' ', sizeof (white));
 		if (I.columns < sizeof (white)) {
```
Comment (radare) [inline libr/cons/cons.c]: this is wrong. just add `if (I.columns > 0 && I.columns < sizeof(white)) {`

Comment (xarkes) [inline libr/cons/cons.c]: fixed

**Comment by radare** [comment]
ill merge after travis, but i guess we should handle when I.columns == 0 and just white[0] = 0;

is this case handled?

**Comment by xarkes** [comment]
Ah right ok, let me fix this

**Pull request merged**